app/views/clinicfinders/_search_form.html.erb

```diff
@@ -1,11 +1,15 @@
 <div class='col-sm-12'>
-  <h5 class='clinic-finder-expand'>
+  <h4 class='clinic-finder-expand'>
     Clinic Locator Assistant
+    <small>This is an experimental feature.</small>
     <span class='glyphicon glyphicon-plus-sign pull-right'></span>
-  </h5>
+  </h4>
+
 
   <div class='col-sm-12 hide' id='clinic-finder-search-form'>
-    <h6>A tool for sorting clinics by cost and distance.</h6>
+    <h5>
+      Filters
```

Comment:
I don't think filters is quite the right verbiage here but I don't know what else to call these fields.

Reply:
"Filter Clinics By: "?

Reply:
love it

Reply:
erred a little on the side of verbosity since this is sort of doing a sort-and-filter, but I think it works either way.